ArticleTemplates/assets/scss/modules/_resizable.scss

```diff
@@ -19,3 +19,11 @@
         }
     }
 }
+
+.resizable_line_height {
```

Comment:
The css generated from this will  apply the appropriate line height to descendant elements of `.resize_line_height` which have the class `line-height-x`.

I can't see from the HTML templates when this would occur as we're never adding the class `line-height-x` to any of the descendant elements of `.resize_line_height`?